### PR TITLE
Splash

### PR DESCRIPTION
**Pull request by robert5974**
## Description: Adding a SplashScreen to the dash application.

**Related issue (if applicable):** fixes #<issue number goes here>

## Checklist:
  - [ ] The code change is tested and works locally.

### CODE REVIEW

File: src/dash.cpp [hunk 1]
```diff
@@ -6,6 +6,32 @@
 
 int main(int argc, char *argv[])
 {
+     QApplication dash(argc, argv);
```
Comment (rsjudka) [inline src/dash.cpp:9]: looks like something got screwed up with the merge... might want to double check the diff here, looks like your version of the file was "stacked" ontop of the original

File: assets/resources.qrc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <RCC>
     <qresource prefix="/">
         <file>aa_video.qml</file>
+	    	<file>splash.png</file>
```
Comment (rsjudka) [inline assets/resources.qrc]: change to spaces instead of tabs (to keep the spacing consistent)

File: README.md [hunk 3]
```diff
@@ -1,143 +1,41 @@
...
-### Install.sh automated install (Raspberry Pi only)
-First you need to run the executable with the following command:
-
-`./install.sh`
-
-Once installed it will start up, to run afterwards, navigate to bin folder and run:
-
-`./dash`
-
-Video walk through:
```
Comment (rsjudka) [inline README.md:38]: can you leave the link to the video, but maybe just give a disclaimer that the process has changed slightly

Comment (rsjudka) [reply README.md:38]: ia renamed to dash, older ui, etc

Comment (robert5974) [reply README.md:38]: Updated with Video walkthrough...per comments

File: assets/resources.qrc (hunk 2 above)
Comment (robert5974) [reply assets/resources.qrc]: corrected tabs to spaces per comment

File: src/dash.cpp (hunk 1 above)
Comment (robert5974) [reply src/dash.cpp:9]: Corrected the dash.cpp.  Not sure why the merge combined the files but good catch.

**Review by robert5974 — COMMENTED** [review]
Corrected with comments.

**Pull request merged**